exercise_solutions.tex

```diff
@@ -1441,6 +1441,256 @@ \subsection*{Solution to \cref{ex:acconn}}
+$J_n(x)$, is:
+\[ \glue_n(x)^{-1}\ct\glue_n(x)\ct\apfunc{j_{n+1}}{H_n(x)}\ct D_{n+1} \]
+Cancelling we get:
+\[ \apfunc{j_{n+1}}{H_n(x)}\ct D_{n+1} \]
+
+We also use the definition of $J_{\blank}$ in the RHS, and then the computation
+rule of $D_{\blank}$, giving us the equalities:
+\begin{align*}
+  & J_{n+1}(i_n(x))\\
+  &= \glue_{n+1}(i_n(x))\ct\apfunc{j_{n+2}}{H_{n+1}(i_n(x))}\ct D_{n+2}\\
+  &= \glue_{n+1}(i_n(x))\ct\apfunc{j_{n+2}}{H_{n+1}(i_n(x))}\ct
+                                  \glue_{n+1}(\north_{n+1})^{-1}\ct D_{n+1}.
+\end{align*}
+
+So it suffices to show:
+\[ \apfunc{j_{n+1}}{H_n(x)} = \glue_{n+1}(i_n(x))\ct\apfunc{j_{n+2}}{H_{n+1}(i_n(x))}
+  \ct\glue_{n+1}(\north_{n+1})^{-1} \]
+Or equivalently:
+\[ \apfunc{j_{n+1}}{H_n(x)} \ct\glue_{n+1}(\north_{n+1}) =
+  \glue_{n+1}(i_n(x))\ct\apfunc{j_{n+2}}{H_{n+1}(i_n(x))} \]
+
+But we remember that we have the homotopy
+$\glue_{n+1} : j_{n+1} = j_{n+2}\circ i_{n+1}$, so, by a simple application
+of \cref{lem:htpy-natural} and the functoriality of $\apfunc{}{}$, we get
+a proof of the equality:
+\[ \apfunc{j_{n+1}}{H_n(x)} \ct\glue_{n+1}(\north_{n+1}) =
+  \glue_{n+1}(i_n(x))\ct\apfunc{j_{n+2}}{\apfunc{i_{n+1}}{H_n(x)}} \]
+So we reduced the goal to showing:
+\[ \apfunc{i_{n+1}}{H_n(x)} = H_{n+1}(i_n(x)) \]
+
+This can be done easily by induction in $\Sn^n$ using the definition of $H_{\blank}$.
+
+
+\subsection*{Solution to \cref{ex:contr-infinity-sphere-susp}}
+
+First we write down the type of the induction principle explicitly:
+
+\[ \ind{\Sn^\infty} : \prd{C:\Sn^\infty \to \UU}{n:C(\north)}{s:C(\south)}
+  \Parens{\prd{x:\Sn^\infty} C(x) \to \dpath C {\merid(x)} {n}{s}} \to \prd{x:\Sn^\infty} C(x) . \]
+
+  We take $\north$ as center of contraction. So we have to prove
+  $\prd{x:\Sn^\infty} \north = x$. For this we use induction on $\Sn^\infty$ taking:
+\[C \defeq (\lambda x. \north = x) : \Sn^\infty \to \UU .\]
+
+  When $x$ is $\north$ we just use $\refl{\north} : \north = \north$. When $x$
+  is $\south$ we use $\merid(\north) : \north = \south$.
+  When $x$ varies along $\merid$ we have to give a function of type:
+\[\prd{x:\Sn^\infty} \Parens{\north = x} \to \Parens{\dpath C {\merid(x)} {\refl{\north}}{\merid(\north)}}. \]
+  So, given $x : \Sn^\infty$ and $p : \north = x$, we have to prove:
+  \[\transfib{x \mapsto (\north = x)}{\merid(x)}{\refl{\north}} = \merid(\north).\]
+
+By \cref{cor:transport-path-prepost} it suffices to show
+$\refl{\north}\ct \merid(x) = \merid(\north)$. Canceling $\refl{\north}$ and
+applying $\merid$ to $p$ gets us the desired result.
+
+\subsection*{Solution to \cref{ex:unique-fiber}}
+
+We know that every two points $y_1,y_2 : Y$ are merely equal, because $Y$ is
+is connected. That is, we have a function
+$c : \prd{y_1,y_2:Y}\trunc {} {y_1 = y_2}$. This is by definition, if we use
+the definition stated in \textit{Remark $3.11.2$}. Otherwise we can use 
+\cref{ex:connectivity-inductively}.
```

Comment:
Ah!  I think the intended definition was that of \cref{ex:connectivity-inductively}, which is the standard meaning of "connencted" (= 0-connected).  The other is actually something that I think now should be changed; will open a new PR.